web/package.json

```diff
@@ -45,7 +45,7 @@
     "@types/node": "^18.7.13",
     "@types/react": "^18.0.17",
     "@types/react-dom": "^18.0.6",
-    "@types/react-helmet": "5",
+    "@types/react-helmet": "^6.1.5",
```

Comment:
upgraded as part of troubleshooting issue. not necessary for this change but it gets us on the latest version